src/components/icon/icon.test.js

```diff
@@ -0,0 +1,237 @@
+import * as React from "react";
+import Icon from "./icon.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { icon, getDataElementByValue } from "../../../cypress/locators";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const colorData = [
+  "rgb(255, 156, 75)",
+  "rgb(205, 56, 75)",
+  "rgb(0, 0, 0)",
+  "rgb(105, 61, 57)",
+];
+
+const IconComponent = ({ ...props }) => {
+  return <Icon type="add" tooltipVisible {...props} />;
+};
+
+const IconTooltipComponent = ({ ...props }) => {
+  return (
+    <div
+      style={{
+        marginLeft: "300px",
+        marginRight: "64px",
+        marginTop: "64px",
+        marginBottom: "64px",
+      }}
+    >
+      <Icon
+        type="add"
+        tooltipVisible
+        tooltipMessage="Hey I'm a tooltip with a different position!"
+        {...props}
+      />
+      ;
+    </div>
+  );
+};
+
+context("Tests for Icon component", () => {
+  describe("should check Icon component properties", () => {
+    it.each(["error", "add", "admin", "alert"])(
+      "should check %s type for Icon component",
```

Comment:
Hi Divya, just a question on this please. In Storybook there is a long list of supported icon types. Is it enough for us to just check 4 icon types here or should we check all supported types?  Perhaps all types are tested by Chromatic in which case our coverage here is sufficient?